Gemfile

```diff
@@ -25,7 +25,6 @@ gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 
 gem "http"
 
-gem "aserto"
```

Comment:
this is not needed as it comes as a dependency of `aserto-rails` https://github.com/aserto-dev/aserto-rails/blob/v2/aserto-rails.gemspec#L34